test/check-dashboard

```diff
@@ -141,9 +141,9 @@ class TestDashboard(MachineCase):
         b.leave_page()
         b.wait_js_cond('$("#machine-dropdown ul li").length == 2')
         b.click('#machine-dropdown ul li:nth-child(1) a')
-        b.wait_js_cond('cockpit.location.href == "/@localhost"')
+        b.wait_js_cond('window.location.hash == "#/@localhost"')
```

Comment:
'cockpit' is only valid inside the private scope ... it's no longer a global on pages that use AMD ... such as the index page.